<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 209, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 181, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 742, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 840, in lint_file
    recipe = _recipe.Recipe.from_file(
        recipe_fname=str(meta_yaml),
    ...<2 lines>...
        renderer=RendererType.RUAMEL,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 245, in from_file
    raise exc from exc
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 236, in from_file
    recipe._load_from_string(text.read())
    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 172, in _load_from_string
    self.render()
    ~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 297, in render
    self.meta = renderer_utils.render(self.recipe_file, self.dump(), self.selector_dict, self.renderer)
                ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/_renderer.py", line 269, in render
    return ruamel.load(yaml_text.replace("\t", " ").replace("%", " "))
           ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/main.py", line 451, in load
    return constructor.get_single_data()
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/constructor.py", line 114, in get_single_data
    node = self.composer.get_single_node()
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 72, in get_single_node
    document = self.compose_document()
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 95, in compose_document
    node = self.compose_node(None, None)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 130, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 211, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 130, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/composer.py", line 204, in compose_mapping_node
    while not self.parser.check_event(MappingEndEvent):
              ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/parser.py", line 141, in check_event
    self.current_event = self.state()
                         ~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/parser.py", line 621, in parse_block_mapping_key
    self.move_token_comment(token)
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/parser.py", line 815, in move_token_comment
    token.move_old_comment(self.scanner.peek_token() if nt is None else nt, empty=empty)
                           ~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/scanner.py", line 1747, in peek_token
    self.fetch_more_tokens()
    ~~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/scanner.py", line 217, in fetch_more_tokens
    self.stale_possible_simple_keys()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/ruamel/yaml/scanner.py", line 348, in stale_possible_simple_keys
    raise ScannerError(
    ...<4 lines>...
    )
ruamel.yaml.scanner.ScannerError: while scanning a simple key
  in "<unicode string>", line 74, column 1:
    {  set tests_to_skip = tests_to_ ...
    ^ (line: 74)
could not find expected ':'
  in "<unicode string>", line 75, column 1:
    {  set tests_to_skip = tests_to_ ...
    ^ (line: 75)

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_92ozpxv5lr/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,5 +1,5 @@
 {% set name = "pyspnego" %}
-{% set version = "0.9.2" %}
+{% set version = "0.11.2" %}
 
 
 package:
@@ -7,32 +7,42 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 0e16cb331f1bd9c85f9b80e2be04bb5da2ecaff678cc0342f68b45e4667d7966
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 994388d308fb06e4498365ce78d222bf4f3570b6df4ec95738431f61510c971b
 
 build:
-  number: 1
+  number: 0
+  skip: True # [py<38]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - pyspnego-parse = spnego.__main__:main
 
 requirements:
-  build:                       # [win]
-    - {{ compiler('c') }}      # [win]
   host:
     - python
-    - cython >=0.29.29,<3.0.0  # [win]
     - pip
     - setuptools
-    - wheel
   run:
     - python
     - cryptography
+    - sspilib >=0.1.0  # [win]
     # Optional dependency for pyspengo-parse, see https://github.com/jborean93/pyspnego#optional-requirements
     - ruamel.yaml
   run_constrained:
     - python-gssapi >=1.6.0  # [not win]
     - pykrb5 >=0.3.0         # [not win]
+
+
+# Windows fails those tests
+{% set tests_to_skip = "test_negotiate_through_python_ntlm" %}
+{% set tests_to_skip = tests_to_skip + " or test_negotiate_with_ntlm_hash" %}
+{% set tests_to_skip = tests_to_skip + " or test_sspi_ntlm_auth_no_sign_or_seal" %}
+{% set tests_to_skip = tests_to_skip + " or test_sspi_ntlm_auth" %}
+{% set tests_to_skip = tests_to_skip + " or test_sspi_ntlm_lm_compat" %}
+{% set tests_to_skip = tests_to_skip + " or test_credssp_multiple_creds" %}
+{% set tests_to_skip = tests_to_skip + " or test_ntlm_iov_wrapping" %}
+{% set tests_to_skip = tests_to_skip + " or test_ntlm_iov_unwrapping_as_stream" %}
+{% set tests_to_skip = tests_to_skip + " or test_sspi_wrap_no_encryption" %}
 
 test:
   source_files:
@@ -40,17 +50,15 @@ test:
   imports:
     - spnego
     - spnego._ntlm_raw
-    - spnego._sspi_raw  # [win]
   requires:
     - pip
     - pytest
     - pytest-mock
+    - sspilib >=0.1.0  # [win]
   commands:
     - pip check
-    - pytest -v  # [unix]
-    # Skip some tests on win64 because of the error
-    # "spnego.exceptions.InvalidTokenError: SpnegoError (9): The logon attempt failed"
-    - pytest -v -k "not (test_ntlm_iov_wrapping or test_ntlm_iov_unwrapping_as_stream or test_sspi_wrap_no_encryption)" --ignore tests/test_auth.py # [win]
+    - pytest tests -v  # [not win]
+    - pytest tests -v -k "not ({{ tests_to_skip }})"  # [win]
 
 about:
   home: https://github.com/jborean93/pyspnego

